docs/administration/authentication-providers/index.md

```diff
@@ -38,20 +38,16 @@ In versions up to 3.5, only a single Authentication Provider could be enabled at
 
 Let's consider that we have UsernamePassword enabled and we create some users, and we've set their email address to their Active Directory domain email address.  The user's can now log in with the username and password stored against their User record.  If we now enable the Active Directory authentication provider, then the users can authenticate using either their original username and password, or they can use a username of user@domain or domain\user along with their domain password, or they can use the Integrated authentication button.  In the first scenario they are actually logging in via the UsernamePassword provider, in the latter 2 scenarios they are using the Active Directory provider, but in all of the cases they end up logged in as the same user (this is the driver behind the fallback checks described in the next section).
```

Comment:
The user's can now log in with the username and password stored against their **User** record

user has a capital letter